docs/src/use/configure/migration-guide.md

```diff
@@ -18,7 +18,7 @@ For reference information on these configuration formats, refer to the following
 
 ## Start Using Flat Config Files
 
-Starting with ESLint v9.0.0, the flat config file format will be the default configuration file format. Once ESLint v9.0.0 is released, you can start using the flat config file format without any additional configuration.
+The flat config file format has been the default configuration file format since ESLint v9.0.0. You can start using the flat config file format without any additional configuration.
 
 To use flat config with ESLint v8, place a `eslint.config.js` file in the root of your project **or** set the `ESLINT_USE_FLAT_CONFIG` environment variable to `true`.
```

Comment:
@nzakas I'm not sure if this belongs in this PR, but it's unclear how to use flat config in v8&mdash;how is placing an `eslint.config.js` in the project root any different than v9 (does `.eslintrc.js` take precedence)? In what circumstances might you want to set `ESLINT_USE_FLAT_CONFIG`? If you're using flat config, wouldn't is be detected because of the `eslint.config.js` already present?

(I didn't edit that part of the docs because I'm not familiar with the specifics of configs.)

Reply:
I think those questions are already answered here:
https://eslint.org/docs/v8.x/use/configure/configuration-files-new